### ISSUE TITLE
Hide unused paths when preset is loaded

### ISSUE THREAD

**Issue by maikito26** [dbagchee/helix-preset-viewer#36]
This would probably serve HX Effects/HX Stomp/Potential future products in Helix family better than the Helix Floor/LT.  However, just to clean up, I'd like to see unused preset paths hidden!  Feel free to close if you disagree.

**Comment by dbagchee** [comment]
Yes I agree - I will also tighten up the display of the paths a little so they don't take up as much horizontal space - essentially trying to mimic hx edit as close as possible :)
